File: app/src/main/java/com/example/musicapplication_sp/model/ClientID.java
package com.example.musicapplication_sp.model;

public class ClientID {
    private String clientID;
    public String iv;

    public ClientID(String clientID, String iv) {
        this.clientID = clientID;
        this.iv = iv;
    }

    public String getClientId() {
        return clientID;
    }
    public void setClientID(String clientId) {
        this.clientID = clientId;
    }

    public String getIv() {
        return this.iv;
    }

    public void setIv(String iv) {
        this.iv = iv;
    }

}

File: app/src/main/java/com/example/musicapplication_sp/model/ClientID.kt
package com.example.musicapplication_sp.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class ClientID(
    @SerializedName("ClientID")
    val ClientID: String?,
    @SerializedName("iv")
    val iv: String?
) : Parcelable {
    constructor() : this("", "")
}
